From 0935efef2a520d351727ccc167a42a980ea142b5 Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Fri, 9 Feb 2024 14:36:36 +0100
Subject: [PATCH 0941/1141] arm64: dts: st: update dsi & ltdc compatible on
 stm32mp251

Add a specific compatible for MP25 SOC.

Change-Id: I49f77755faf5d677e417856d876866a1b0a98c75
Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/358851
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp251.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/st/stm32mp251.dtsi b/arch/arm64/boot/dts/st/stm32mp251.dtsi
index 6d9d8e84c77f..a357891bc12b 100644
--- a/arch/arm64/boot/dts/st/stm32mp251.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp251.dtsi
@@ -2061,7 +2061,7 @@ i3c4: i3c@46080000 {
 			};
 
 			dsi: dsi@48000000 {
-				compatible = "st,stm32-dsi";
+				compatible = "st,stm32mp25-dsi";
 				reg = <0x48000000 0x800>;
 				#clock-cells = <0>;
 				clocks = <&rcc CK_BUS_DSI>, <&rcc CK_KER_DSIPHY>,
@@ -2075,7 +2075,7 @@ dsi: dsi@48000000 {
 			};
 
 			ltdc: display-controller@48010000 {
-				compatible = "st,stm32-ltdc";
+				compatible = "st,stm32mp25-ltdc";
 				reg = <0x48010000 0x400>;
 				st,syscon = <&syscfg>;
 				interrupts = <GIC_SPI 158 IRQ_TYPE_LEVEL_HIGH>,
-- 
2.39.2

